template_debian/packages_minimal.list

```diff
@@ -3,6 +3,4 @@ sudo
 dmsetup
 psmisc
 gnupg
-xterm
-libfile-mimeinfo-perl
```

Comment:
This may require adding it to documentation for minimal template - from what I see, it's one of the options that `xdg-open` uses (which we do use), and if you start with minimal template, it's probably the only option. So, without this package opening files from other qubes (for example via `qvm-open-in-vm`) may stop working.

Reply:
Interesting, then thats as well should be add it to [qubes-vm-recommended](https://github.com/QubesOS/qubes-meta-packages/blob/main/debian/control#L31), because it has usefulness behind it. (not sure if it possible (easily) to rewrite this functionality into a different better language instead)

Reply:
The problem (similar to xterm) is that `libfile-mimeinfo-perl` is just one of the backends `xdg-open` can use (it's the fallback one, if none other is found). Normally `xdg-open` tries to use one for matching environment (`exo` for Xfce, `gnome-open` for Gnome etc). I'd rather avoid listing all possible alternatives in package dependencies (that will make template customization harder), and I don't see virtual name for this feature in Debian.

Reply:
I see, then we can keep this.